apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/GrpcRdaSource.java

```diff
@@ -135,12 +151,28 @@ public int retrieveAndProcessObjects(int maxPerBatch, RdaSink<TResponse> sink)
       }
     }
     if (interrupted) {
-      LOGGER.warn("interrupted with processedCount {}", processed);
+      LOGGER.warn("{} claim processing interrupted with processedCount {}", claimType, processed);
     }
     metrics.successes.mark();
     return processed;
   }
 
+  /**
+   * Uses the hard coded starting sequence number if one has been configured. Otherwise asks the
+   * sink to provide its maximum known sequence number as our starting point. When all else fails we
+   * start at the beginning.
+   *
+   * @param sink used to obtain the maximum known sequence number
+   * @return a valid RDA change sequence number
+   */
+  private long getStartingSequenceNumber(RdaSink<TResponse> sink) throws ProcessingException {
+    if (startingSequenceNumber.isPresent()) {
+      return startingSequenceNumber.get();
+    } else {
+      return sink.readMaxExistingSequenceNumber().map(seqNo -> seqNo + 1).orElse(MIN_SEQUENCE_NUM);
```

Comment:
Why `seqNo + 1`?

Reply:
If the last one we wrote to the database has sequence 100 then we don't need to receive that record again.  So to avoid the repeat we ask for 101.

Reply:
ah of course, I'm not sure what I was thinking